Wording & spelling fixes (#221)
Kattis/problem-package-format: spec/2023-07-draft.md

| uuid                   | String                                                    |                                                          | Required. UUID identifying the problem.
| version                | String                                                    | Version of the problem described in the package.         |
| credits                | String or map with keys as defined below.                 |                                                          | Who should get credits for creating this problem. See below for details.
| source                 | String, a sequence, or a map as [defined below](#source). |                                                          | The problem set from which the problem originates. See below for details.
| license                | String                                                    | unknown                                                  | License under which the problem may be used. Value has to be one of the ones defined below.

> | rights_owner           | String                                                    | Value of credits, if present, otherwise value of source. | Owner of the copyright of the problem. If not present, authors are the owners. If author is not present either, source is owner. Required if license is something other than `unknown` or `public domain`. Forbidden if license is `public domain`.

Reviewer: I think this should specifically be the `authors` key of `credits`.

> | rights_owner           | String                                                    | Value of credits.authors, if present, otherwise value of source. | Owner of the copyright of the problem. If not present, authors are the owners. If author is not present either, source is owner. Required if license is something other than `unknown` or `public domain`. Forbidden if license is `public domain`.

Reviewer: ```suggestion
| rights_owner           | String                                                    | Value of `credits.authors`, if present, otherwise value of `source`. | Owner of the copyright of the problem. Required to have a value if license is something other than `unknown` or `public domain`. Forbidden if license is `public domain`.
```
